src/TelemetryChannels/ServerTelemetryChannel/Shared/AdaptiveSamplingTelemetryProcessor.cs

```diff
@@ -71,6 +71,16 @@ public string ExcludedTypes
         }
 
         /// <summary>
+        /// Gets or sets a semicolon separated list of telemetry types that should be sampled. All types are sampled when left empty.
+        /// </summary>
+        public string IncludedTypes
+        {
+            get { return this.samplingProcessor.IncludedTypes; }
```

Comment:
Unit test for this is here: https://github.com/Microsoft/ApplicationInsights-dotnet/pull/259/commits/ef7c0d45ba5a16b027ee0928a2d51e548f2cf86c
Added a bunch of tests for new stuff, old tests the cover rest.

Fixed the description.